poc/shared_infrastructure/variables-network.tf

```diff
@@ -2,12 +2,21 @@
 # Network variables #
 #####################
 
-# environment
 variable "environment" {
   type        = string
   description = "This variable defines the environment to be built"
 }
 
+variable "location" {
+  type        = string
+  description = "Define the primary Azure region"
+}
+
+variable "failover_location" {
+  type        = string
+  description = "Define the failover Azure region; Should be geographically separate from the primary location"
+}
```

Comment:
Would we probably use the same failover location for any services/resources that accept one (idk how many this is)?

Reply:
That was my intent.